state-network.md

```diff
@@ -116,39 +116,22 @@ A naive approach to storage of trie nodes would be to simply use the `node_hash`
 We define path as a sequences of "nibbles" which represent the path through the MPT to reach the trie node.
 
 ```
-nibble := {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, a, b, c, d, e, f} // 16 possible values
-NibblePair := Byte // 2 nibbles tightly packed into a single byte
-Nibbles := List(NibblePair, max_length=32) // fixed path length of 8 bytes
+nibble     := {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, a, b, c, d, e, f}
+NibblePair := Byte  # 2 nibbles tightly packed into a single byte
+Nibbles    := Container(is_odd_length=bool, packed_nibbles=List(NibblePair, max_length=32))
```

Comment:
This approach to serialization is intentionally simplistic.  There are more optimal ways that we could encode things to try and remove the variable length object... but for now this should be an adequate starting point.